health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccess.js

```
@@ -316,6 +323,43 @@ function UserAccess({ category, setData, nationalRoles }) {
+              label={t("YES")}
+              onClick={() => {
+                handleUpdateAssignEmployee(unassignPopup);
+              }}
+            />,
+            <Button type={"button"} size={"large"} variation={"secondary"} label={t("NO")} onClick={() => { setUnassignPopup(false); }} />,
+          ]}
+          sortFooterChildren={true}
+          onClose={() => {
+            setUnassignPopup(false);
+          }}
+        ></PopUp>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)